Tracker/Controllers/NewTrackerViewController.swift

```diff
@@ -239,13 +265,16 @@ extension NewTrackerViewController: UITableViewDelegate {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         switch indexPath.row {
         case 0: // "Категория"
-            let categoriesVC = CategoriesViewController()
-            categoriesVC.delegateTransition = self
-            categoriesVC.checkmarkedCell = lastCategory
+            guard let categoriesVC = SomeCoordinator.start(
+                with: SomeConfiguration(
+                    lastCategory: lastCategory ?? nil)
+            ) as? CategoriesViewController
+            else { return }
```

Comment:
Нужно стараться держать код в консистентном состоянии и не смешивать подходы/стили.
Посмотри, ты одну круглую скобку на новую строчку перенес, а другую оставил на той же строке

Это не последовательно, и потому код как будто бы в один пельмень слипается, разобрать тяжело)

```suggestion
            guard 
                let categoriesVC = SomeCoordinator.start(
                    with: SomeConfiguration(lastCategory: lastCategory)
                ) as? CategoriesViewController
            else { return }
```